src/psyclone/tests/psyir/transformations/intrinsics/matmul2code_trans_test.py

```diff
@@ -494,12 +494,12 @@ def test_validate10():
     matrix.children[0] = Literal("1", INTEGER_TYPE)
     with pytest.raises(TransformationError) as excinfo:
         trans.validate(matmul)
-    assert ("To use matmul2code_trans on matmul, exactly two indices of the "
-            "1st argument 'x' must be full ranges "
-            "but found 1." in str(excinfo.value))
+    assert (f"To use {trans.name} on matmul, exactly two indices of the "
```

Comment:
Please could you change this so that it checks that the output string is what we expect, rather than what we coded. i.e. this doesn't need to be an f-string - just hardwire in the values that we expect.

Reply:
Addressed in c4d60bc3ef52a6d7b2853dd5b3c3b01ce82996dc.